src/hypothesis/strategies.py

```diff
@@ -1015,13 +1012,14 @@ def fractions(min_value=None, max_value=None, max_denominator=None):
     be None or a positive integer.
 
     """
+    check_valid_bound(min_value, 'min_value')
```

Comment:
Thinking about this also lead to my realisation of #814 which I think is another argument that forcing a bit better testing of our validation is a good thing!